content/about/releases.pt-br.md

```diff
@@ -0,0 +1,15 @@
+---
+title: releases
+displayTitle: Versões
+description: 'Versões | Node.js'
+authors: ZYSzys, nstanard,  mikeal, fhemberger, garybernhardt, piepmatz, boneskull, bjb568, ItaloPussi
+category: about
+---
+
+Uma atualização major do Node.js tem suporte total durante seis meses, 
+tempo que permite que bibliotecas terceiras sejam atualizadas. Depois desse período, versões impares (9, 11, etc.) se tornam obsoletas e versões pares (10, 12, etc.)
+entram em _LTS_ e estão prontas para uso geral. Versões _LTS_ possuem um tempo de suporte 
+de 30 meses cujo garante a correção de BUGs críticos nesse período. Aplicações 
+produtivas somente devem utilizar versões _LTS_.
```

Comment:
What do you think about replacing "produtivas" (productive) to "de produção" (production)?